mkdocs.yml

```diff
@@ -67,6 +67,10 @@ nav:
       - Local Files: tutorials/data-loading/local-files.md
       - Remote Files: tutorials/data-loading/remote-files.md
       - Databases: tutorials/data-loading/databases.md
+  - Tooling:
+    - Overview: tooling/overview.md
+    - corectl: tooling/corectl.md
+    - Catwalk: tooling/catwalk.md  
```

Comment:
`Catwalk` or `catwalk`? I've only seen `catwalk` elsewhere

Reply:
It should be ```catwalk```